src/common/libkvs/treeobj.c

```diff
@@ -359,6 +359,23 @@ json_t *treeobj_create_val (const void *data, int len)
     return obj;
 }
 
+json_t *treeobj_create_val_base64 (const char *data)
+{
+    json_t *obj = NULL;
+
+    if (!data)
```

Comment:
I see your point, a NULL to ```treeobj_create_val_base64()``` doesn't make sense compared to a NULL & length 0 to ```treeobj_create_val()```.

Yeah, ```treeobj_create_val()``` can be refactored.